bskyogcard/src/data/getImage.ts

```diff
@@ -0,0 +1,12 @@
+import MIMEType from 'whatwg-mimetype'
+
+export async function getImage(url: string) {
+  const response = await fetch(url)
+  const mimeType = new MIMEType(response.headers.get('content-type') ?? '')
+  const arrayBuf = await response.arrayBuffer() // must drain body even if it will be discarded
+  if (response.status !== 200) return null
+  return {
+    mime: mimeType.essence as string | undefined,
+    image: Buffer.from(arrayBuf),
+  }
+}
```

Comment:
Extended what was already in this project to check the mime type. I needed this for SP images, which are PNGs, whereas our other images are JPEGs afaict. May be good in the future to support whatever mime is specified by the responding server.